controllers/topicsController.js

```diff
@@ -1,14 +1,14 @@
-const { fetchTopics, fetchReview } = require("../models/categoriesModels");
+const { fetchTopics, fetchArticleId } = require("../models/categoriesModels");

```

Comment:
Again would fetch article by id not have it's own articleModel? We would expect to separate out different data sets